src/client/routes/dashboard/Dashboard.tsx

```diff
@@ -1,17 +0,0 @@
-import React, { FunctionComponent } from 'react';
-import { Switch, Route } from 'react-router-dom';
-import HackerDash from './HackerDash';
-import OrganizerDash from './OrganizerDash';
-
-export const Dashboard: FunctionComponent = (): JSX.Element => {
```

Comment:
Did you replace this somewhere else?

Reply:
This component wasn't used in the actual rendering of the app because it has a security hole: any hacker could access the organizer dash just by navigating to `/dashboard/organizer`